pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -57,38 +57,38 @@ func NewWatcher(sockDir string) Watcher {
 }
 
 // AddHandler registers a callback to be invoked for a particular type of plugin
-func (w *Watcher) AddHandler(handlerType string, handlerCbkFn RegisterCallbackFn) {
+func (w *Watcher) AddHandler(pluginType string, handlerCbkFn RegisterCallbackFn) {
 	w.mutex.Lock()
 	defer w.mutex.Unlock()
-	w.handlers[handlerType] = handlerCbkFn
+	w.handlers[pluginType] = handlerCbkFn
 }
 
 // Creates the plugin directory, if it doesn't already exist.
 func (w *Watcher) createPluginDir() error {
 	glog.V(4).Infof("Ensuring Plugin directory at %s ", w.path)
 	if err := w.fs.MkdirAll(w.path, 0755); err != nil {
-		return fmt.Errorf("error (re-)creating driver directory: %s", err)
+		return fmt.Errorf("error (re-)creating root %s: %s", w.path, err)
```

Comment:
Hi @figo,

Consider using `%v` when formatting an `error` type as the `error` interface forces `Error() string`, not `String() string`, and thus `%s` may not actually present data as nicely as `%v` in this case.

Reply:
thanks